bitnami/tomcat/templates/_pod.tpl

```diff
@@ -115,6 +119,30 @@ containers:
 {{- if .Values.sidecars }}
 {{ include "common.tplvalues.render" ( dict "value" .Values.sidecars "context" $) }}
 {{- end }}
+{{- if .Values.metrics.jmx.enabled }}
+- name: jmx-exporter
+  image: {{ template "tomcat.metrics.jmx.image" . }}
+  imagePullPolicy: {{ .Values.metrics.jmx.image.pullPolicy | quote }}
+  command:
+    - java
+    - -XX:+UnlockExperimentalVMOptions
+    - -XX:+UseCGroupMemoryLimitForHeap
+    - -XX:MaxRAMFraction=1
+    - -XshowSettings:vm
+    - -jar
+    - jmx_prometheus_httpserver.jar
+    - "5556"
+    - /etc/jmx-tomcat/jmx-tomcat-prometheus.yml  
+  ports:
+    - name: {{ .Values.metrics.jmx.portName | default "metrics" }}
+      containerPort: {{ .Values.metrics.jmx.port | default "5556" }}	
```

Comment:
The `default` is not needed since those values are already set in the Values.yaml

Reply:
removed